.github/file-paths.yaml

```diff
@@ -29,8 +29,8 @@ frontend_configs: &frontend_configs
   - ".eslintrc.js"
   - "postcss.config.js"
   - "rspack.config.js"
-  - "webpack.config.js"
-  - "webpack.embedding-sdk.config.js"
+  - "rspack.main.config.js"
```

Comment:
Do we have both rspack.config.js and rspack.main.config.js at once?

Reply:
yes. rspack.config.js builds `main` and 2 sdk-related configs
the `main` is for main config for app and that is used as the base in other configs

i had to split because of circular references